source/connection.cc

```diff
@@ -118,6 +118,19 @@ RestClient::Connection::SetUserAgent(const std::string& userAgent) {
 }
 
 /**
+ * @brief set custom Certificate Authority (CA) path
+ * default restclient-cpp/RESTCLIENT_VERSION string
+ *
+ * @param caInfo - The path to a file holding the certificates used to
+ * verify the peer with. See CURLOPT_CAINFO
+ *
+ */
+void
+RestClient::Connection::SetCAInfo(const std::string& caInfo) {
+  this->caInfo = caInfo;
+}
```

Comment:
I considered it, but decided to keep consistent with libcurl naming (even though it is not as clear as I would have liked). If you prefer clarity of names in restclient-cpp even though they would not be consistent with libcurl, I would be happy to change--just commenting to get it considered.